app/views/application/donation.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= render "header" %>
+<br>
+<div class = "donation-link">
+  <h1 class = "padding-top:5px text-center">
```

Comment:
This says class attribute but then specifies style?
Also, style previously has been no whitespace around equal signs, please be consistent.